Source/Core/DiscIO/DirectoryBlob.cpp

```diff
@@ -896,11 +912,12 @@ DirectoryBlobPartition::DirectoryBlobPartition(
   if (sys_callback)
     sys_callback(&sys_nodes);
 
-  SetDiscHeader(ExtractNodeToVector(&sys_nodes, &m_disc_header));
-  SetDiscType(is_wii);
-  SetBI2(ExtractNodeToVector(&sys_nodes, &m_bi2));
+  disc_header = std::move(ExtractNodeToVector(&sys_nodes, &disc_header));
```

Comment:
Unnecessary `std::move`. (This is already an xvalue.)

```suggestion
  disc_header = ExtractNodeToVector(&sys_nodes, &disc_header);
```